api-flask/app/kobo.py

```diff
@@ -202,6 +204,22 @@ def get_responses_from_kobo(
     return form_responses, form_labels
 
 
+def get_form_dates(form_url: HttpUrl, form_name: str, datetime_field: str):
```

Comment:
@laurentS is there an easy way to add timed caching to this function? Say for 6hours?